ui/app/styles/components/auth-buttons.scss

```diff
@@ -20,8 +21,9 @@
 .auth-button-type-gitlab {
   position: relative;
   top: -6px;
-  left: -0.75rem;
+  left: -$size-9;
```

Comment:
This is more of an overall thought on variables that came to mind when I looked at this. Say for example we decided that `size-9` was too small and we wanted to bump it up a bit. Would we want the position of this element to change? I'm just wondering if for positions if it's safer to use specific values to ensure that the element stays in place.

Reply:
There are other instances as I'm looking ahead and I won't call them all out but the same goes for padding, margin, height and width as well. This might be a good topic for our css overhaul discussion.

Reply:
I've gone back and forth on this with similar concerns. But I've move forward that it could be helpful to replace everything with the var so we get a sense of where and how often it is used (quick search for say `$size-9`). There is a whole refactor that needs to happen on the var front outside this work. However, for this sidebranch/project do you think I should hold off of replacing previously existing values for vars?

Reply:
It seems maybe a bit out of scope but not sure if it's worth undoing. I suppose if we decide not to use variables for position and spacing then it might be hard to go through later and fall all of these and convert pack to fixed values. It seems unlikely that we would change the size variables but I just wanted to raise the issue so up to you if you want to leave as is.